docs/LogGuide.md

```diff
@@ -861,6 +861,46 @@ There are also a number of examples where tethers are generated in some other wa
 * Suzaku Extreme birbs: who knows
 * player to player tethers (dragonsight, cover, fairy tether)
 
+## 24:LimitBreak
+
+This log line is recorded every server tick where limit break energy is generated while in combat in a light or full party. (Generation is not recorded while at cap.) It starts at 0x0000 at the beginning of the instance (or encounter in the case of a single-encounter instance,) and counts up by 0x00DC (220 decimal) until the limit break is used, or to a max of 0x7530 (30,000 decimal). This rate of increase is constant, but other actions taken can cause extra increments to happen independent of the base increase. (These other increments occur in the same packet as the base rate, but separately.) Each limit break bar is 0x2710 (10,000 decimal) units.
```

Comment:
In github markdown at least, you can break up each sentence (or fragments) onto their own line.  It makes future edits easier to read.  Would you mind inserting some line breaks here?  The formatted markdown is equivalent, but the raw markdown is better.

You can see above near `The reason why there is such a discrepancy between ACT and fflogs` for an example.

Reply:
Certainly. What's your preferred maximum, 110 characters? (I realize I can't just break at "word boundary nearest 110", but having that as a rough guideline is helpful.)

It also looks like this is something my previous additions to the log guide didn't respect, would you prefer that fixed here or in a separate update?

Reply:
It's less about word boundaries and more a desire for semantic line breaks: https://sembr.org/.  If you made me pick a limit, I'd say 80 is probably reasonable for most things?

Also I assume that the max here of 0x7530 is lb3? and that if you only get lb1 or lb2 it stops at lower values?